components/profile/Availability.tsx

```diff
@@ -73,15 +80,19 @@ const Availability = ({
               name="endDate"
               render={({ field }) => (
                 <ReactDatePicker
-                  className={`${textColor} bg-black-700 paragraph-3-regular  w-full border-none py-4 pl-4 pr-1`}
+                  className={`${textColor} react-datepicker__view-calendar-icon paragraph-3-regular h-11 w-full border-none bg-transparent !px-0`}
```

Comment:
We shouldn't need to add the `react-datepicker__view-calendar-icon` class, it's already part of the class added to the datepicker. We're just adding to their styles.

Reply:
<img width="620" alt="Screenshot 2024-03-29 at 9 58 50 AM" src="https://github.com/DDVVPP/GitNote/assets/47732125/d28133a8-c869-480b-bdae-31f934d2b443">

It doesn't seem to work without it...

Reply:
Just to keep track of things in here - 
ReactDatePicker already comes with it's own class `react-datepicker__view-calendar-icon`

I moved the `react-datepicker__view-calendar-icon` class styles outside of the utilities in `globals.css` so that it is built even before tailwind sees the className in the source code.